packages/components/src/internal/components/picklist/AddToPicklistMenuItem.tsx

```diff
@@ -42,7 +43,7 @@ export const AddToPicklistMenuItem: FC<Props> = memo(props => {
     }, []);
 
     const onClick = useCallback(() => {
-        if (queryModel?.hasSelections) {
+        if (queryModel?.hasSelections || sampleIds?.length) {
```

Comment:
I have a similar fix for this issue (which I introduced recently) in my PR. Just a heads up as I'm not sure which of us will merge first.